api-gen/extraction/index.ts

```diff
@@ -1,11 +1,22 @@
 import {writeFileSync} from 'fs';
+import ts from 'typescript';
+// @ts-ignore This compiles fine, but Webstorm doesn't like the ESM import in a CJS context.
```

Comment:
FYI: Might be able to get this working by adding a `package.json` with `type: module`. See `ng-dev/package.json`

Reply:
Ahh, I had tried adding a `tsconfig.ide.json` here and it worked for compiler-cli, but not for `typescript`. I'll give it a shot.

Reply:
I tried adding a `package.json` and a `tsconfig.ide.json`, and that did fix the problems I had where it saw `@angular/compiler-cli` and `typescript` as incompatible, but there was a _different_ problem of it not seeing `CompilerOptions` or `CompilerHost` from the compiler-cli entry point.